cloud-init.yaml

```diff
@@ -0,0 +1,37 @@
+#cloud-config
+
+packages:
+  - podman
+  - podman-compose
+
+snap:
+  commands:
+    0: [install, core]
+    1: [refresh, core]
+    2: [install, --classic, certbot]
+
+write_files:
+  #
+  # This should NOT be updated unless you are intending to redeploy the bootstrap server
+  #
+  - content: |
+      services:
+        bootstrap:
+          image: ghcr.io/holochain/kitsune2_bootstrap_srv@sha256:02e2d138496c000ebb17d4ace84507870b4f5967ed270d814a8a4e312e8c1c83
+          command:
+            - kitsune2-bootstrap-srv
+            - --production
+            - --listen
+            - "[::]:443"
```

Comment:
Note, only binding to `[::]` but dual stack is available so `curl -ksS https://164.92.135.22/health` works fine